spyder/plugins/appearance/widgets.py

```diff
@@ -60,6 +62,41 @@ def get_scheme_name(self):
         """
         return self.scheme_name_textbox[self.last_used_scheme].text()
 
+    def validate_colors(self):
+        """
+        Validate the current color scheme and display a message box listing 
+        any invalid colors.
+        """
+        invalid_colors = {}
+        scheme_name = self.last_used_scheme
+        pattern = r"^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3}|[A-Fa-f0-9]{8}|[A-Fa-f0-9]{4})$"
+        for key in self.widgets[scheme_name]:
+            items = self.widgets[scheme_name][key]
+
+            if(not bool(re.match(pattern, items[0].text()))):
+                    invalid_colors[key] = items[0].text()
+
+        if invalid_colors:
+            message = _("The following properties have invalid colors:\n\n")
+            for property_name, color in invalid_colors.items():
+                name = syntaxhighlighters.COLOR_SCHEME_KEYS[property_name]
+                clean_name = name[:-1].replace("<br>","")
+                message += _(
+                    "The property {} has an invalid color: {}.\n"
+                    ).format(clean_name, color)
+            title = _('Error getting colors.')
+            self.msgbox = QMessageBox(
+                        QMessageBox.Warning,
+                        title,
+                        message,
+                        QMessageBox.Ok,
+                        self
+                    )
```

Comment:
```suggestion

            msgbox = QMessageBox(
                QMessageBox.Warning,
                _('Error setting colors'),
                message,
                QMessageBox.Ok,
                self
            )
```

- No need of an additional variable for the title.
- Fix title name (the error happens when we're setting the colors, not getting them).
- No need to add this message as a class attribute (i.e. no `self.msgbox`)
- Add empty line between the previous `for` and this for clarity.